docs/dev_guide/diagnostic_ref/dpct1023.rst

```diff
@@ -22,4 +22,43 @@ parameter was not used.
 Suggestions to Fix
 ------------------
 
-Verify the code correctness.
\ No newline at end of file
+Verify the code correctness.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+    __global__ void kernel(unsigned mask, int val, int srcLane) {

```

Comment:
suggest to use mask, val, srcLane as local variable

Reply:
I changed the function from __global__ to __device__